### PR TITLE
Update the module with latest syntax

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,23 +49,22 @@ import ballerina/io;
...
-        clientConfig: {}
+ConsulConfiguration consulConfig = {
+     uri: "http://localhost:8500",
+     aclToken: "",
+     clientConfig: {}
 };
+
+Client consulClient = new(consulConfig);
+    
+function main(string... args) {
```
Comment (ldclakmal) [inline README.md]:
```suggestion
public function main() {
```

File: consul/consul_data_mappings.bal [hunk 2]
```diff
@@ -95,15 +95,20 @@ function convertValues(json jsonStatus) returns Value {
 function convertToInt(json jsonVal) returns int {
     string stringVal = jsonVal.toString();
     if (stringVal != "") {
-        return check <int>stringVal;
+        var value = int.create(stringVal);
```
Comment (ldclakmal) [inline consul/consul_data_mappings.bal]: `.create(...)` function is changed as `.convert(...)`
Please replace in all the places.

File: consul/consul_endpoint.bal [hunk 3]
```diff
@@ -17,39 +17,316 @@
...
+        return httpResponse;
+    } else {
+        int statusCode = httpResponse.statusCode;
+        var consulJSONResponse = httpResponse.getJsonPayload();
+        if (consulJSONResponse is error) {
+            return consulJSONResponse;
+        } else {
+            if (statusCode == 200) {
+                checkResponse = convertToHealthClients(<json[]>consulJSONResponse);
+            return checkResponse;
```
Comment (ldclakmal) [inline consul/consul_endpoint.bal]: Please format the code here.

File: consul/consul_endpoint.bal [hunk 4]
```diff
@@ -17,39 +17,316 @@
...
+    if (httpResponse is error) {
+        return httpResponse;
+    } else {
+        int statusCode = httpResponse.statusCode;
+        if (statusCode == 200) {
+            return true;
+        } else {
+                var consulStringResponse = httpResponse.getTextPayload();
+                if (consulStringResponse is error) {
+                return consulStringResponse;
```
Comment (ldclakmal) [inline consul/consul_endpoint.bal]: Please format the code here.

File: consul/consul_endpoint.bal [hunk 5]
```diff
@@ -17,39 +17,316 @@
...
+
+    var httpResponse = self.consulClient->put(consulPath, request);
+    if (httpResponse is error) {
+        return httpResponse;
+    } else {
+        int statusCode = httpResponse.statusCode;
+        if (statusCode == 200) {
+            return true;
+        } else {
+        var consulStringResponse = httpResponse.getTextPayload();
```
Comment (ldclakmal) [inline consul/consul_endpoint.bal]: Please format the code here.

**Review by ldclakmal — APPROVED** [review]

**Review by shafreenAnfar — APPROVED** [review]

**Pull request merged**